lib/daru/category.rb

```diff
@@ -14,7 +14,7 @@ def initialize_category data, opts={}
       if opts[:categories]
         validate_categories(opts[:categories])
         add_extra_categories(opts[:categories] - categories)
-        self.order = opts[:categories]
+        set_order opts[:categories]
```

Comment:
I've set `order=` (now `set_order`) as private. There's another function `categories=` which takes in order of categories and if there are some categories that do not exist then those are added too. Since `categories=` also implements the function of `order=` I removed it from public access.